recipe/build-core.sh

```diff
@@ -4,12 +4,17 @@ set -xe
 bazel clean --expunge
 bazel shutdown
 
+if [[ "${target_platform}" == osx-64 ]]; then
+  # Fix "too many open files" error
+  ulimit -s 65532
```

Comment:
A moot point but:

```suggestion
  ulimit -s $(ulimit -Hs)
```

which sets it to whatever the hard limit is _on that box_.  On my box the hard limit is 65520.  YMMV.

Reply:
This isn't even executed. Worth another 10+ hours of builds to  just remove these lines. Or would you be fine if I mark it in the ticket for 2.48?

Reply:
Yeah, it _did_ make my MBP sweat for a while when I built it locally... 🤔 

I think we're agreed that we'll not build `osx-64` here so, make a note for next time.

However, I was concerned that the magic number might have been wrong.
That said, I've just looked on two Intel macs _chez Ian_ and they both have 65532 and my own ARM64 mac shows 65520 (like the Anaconda MBP).
So there's an AMD/ARM difference and your 65532 looks right for `osx-64`.  (But is still a magic number.)